### ISSUE TITLE
Backslash \ in password is removed during conversion; hilarity results

### ISSUE THREAD

**Issue by NemanSyed** [boredwookie/tools#2]
**TL;DR: Please add two 'gotcha' items to the blog entry re: \ chars in passwords and lists.**

Hi! Thanks for this fantastic piece of software! My BlackBerry Password Keeper successfully exported a password as such as the CSV:
`6}\A$^DL\rwY.U/G`
Note the two \ characters.

Post-conversion, it's rendered properly as such in the CSV:
`"6}\A$^DL\rwY.U/G"`
Backslashes are retained; whole string is in quotes. The order of course, is different from the Password Keeper export.

After import to KeePass 2.49, the \ characters are lost:
`6}A$^DLrwY.U/G`

In this example, the password is simply wrong. In another password's case, where the \ was at the end, KeePass' import not only loses the \ character, but completely misinterprets all following imports. You might as well throw away the DB. (Interestingly, it preserves the closing quote and comma, right up until the _opening_ quote of the notes following the URL [which follows the password], so the imported password is finally `password",URL,`.

The removal only happened in the password field as far as I can tell. It's preserved in the notes. It's currently unknown whether any other characters get eaten. For example, single quote `'`, double quote `"`, and backtick ` all survive.

This is due to KeePass, but would you be so kind as to put a warning in the "gotchas" section of https://7thzero.com/blog/bb-password-keeper-to-keepass-tool to consider either checking passwords before import or disabling the **Interpret '\\' as an escape character** option when setting up the import? Maybe also identify lists are partially converted therefore not imported as one might expect. Thanks!
